src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -0,0 +1,838 @@
+<!-- eslint-disable vue/no-v-html -->
+<template>
+  <div
+    :class="{ 'set-min-width': hasTimePeriods }"
+    :style="widthStyle"
+    class="k-datetime-picker"
+  >
+    <KPop
+      :hide-popover="hidePopover"
+      class=""
+      placement="bottomStart"
+      width="auto"
+      hide-caret
+      position-fixed
+      @closed="handleClose"
+    >
+      <KButton
+        :class="{ 'set-min-width': hasTimePeriods }"
+        :is-rounded="false"
+        :style="widthStyle"
+        size="large"
+        class="timepicker-input"
```

Comment:
Let's add an `aria-role` here
```suggestion
        aria-role="input"
        class="timepicker-input"
```